astropy/cosmology/tests/test_flrw.py

```diff
@@ -12,15 +12,33 @@
 import astropy.units as u
 from astropy.cosmology import (FLRW, FlatLambdaCDM, Flatw0waCDM, FlatwCDM,
                                LambdaCDM, Planck18, w0waCDM, w0wzCDM, wCDM, wpwaCDM)
+from astropy.cosmology.flrw import ellipkinc, hyp2f1, quad
+from astropy.utils.compat.optional_deps import HAS_SCIPY
 
-from .test_core import FlatCosmologyMixinTest
 from .test_core import CosmologySubclassTest as CosmologyTest
+from .test_core import FlatCosmologyMixinTest
 
 ##############################################################################
 # TESTS
 ##############################################################################
 
 
+@pytest.mark.skipif(HAS_SCIPY, reason="can't have :mod:`scipy` imported")
```

Comment:
Shouldn't the `reason` be something like "Cannot test exception with :mod:`scipy` imported"?